10.js

```diff
@@ -0,0 +1,23 @@
+const resolveSymlinks = input => {
+  const inputArray = input.split('\n')
+  let symlinks = []
+  let path = inputArray[inputArray.length - 1]
+
+  for (let index = 1; index <= inputArray[0]; index++) {
+    symlinks.push(inputArray[index])
+  }
+
+  symlinks = symlinks.map(symlink => symlink.split(':'))
+
+  symlinks.forEach(symlink => {
+    const symlinkRegExp = new RegExp(symlink[0])
```

Comment:
I recommend declaring this *outside* of the loop - this approach (assuming nothing optimizes this code) means we re-declare the regular expression object in each iteration of the loop.